docs/advanced_topics/accessibility_considerations.md

```diff
@@ -73,14 +73,14 @@ If this is a concern to you, you can change which tags are used when saving cont
 
 ### TableBlock
 
-The [TableBlock](../reference/contrib/table_block) default implementation makes it too easy for end-users to miss they need either row or column headers ([#5989](https://github.com/wagtail/wagtail/issues/5989>)). Make sure to always have either row headers or column headers set.
+The [TableBlock](../reference/contrib/table_block) default implementation makes it too easy for end-users to miss. They need either row or column headers ([#5989](https://github.com/wagtail/wagtail/issues/5989>)). Make sure to always have either row headers or column headers set.
```

Comment:
The sentence would make no sense with this change included! It’s the need for row and column headers that people will miss.

```suggestion
The [TableBlock](../reference/contrib/table_block) default implementation makes it too easy for end-users to miss they need either row or column headers ([#5989](https://github.com/wagtail/wagtail/issues/5989>)). Make sure to always have either row headers or column headers set.
```